internal/winapi/utils.go

```diff
@@ -36,7 +45,7 @@ func NewUnicodeString(s string) (*UnicodeString, error) {
 		MaximumLength: uint16(len(ws) * 2),
 		Buffer:        &make([]uint16, len(ws))[0],
 	}
-	copy((*[32768]uint16)(unsafe.Pointer(uni.Buffer))[:], ws)
+	copy(Uint16BufferToSlice(uni.Buffer, len(ws)), ws)
```

Comment:
I made a recent PR here https://github.com/microsoft/hcsshim/pull/918 that changes the logic of this function, in particular this line. It was just merged today, sorry for the inconvenience here!

Reply:
No worries, looks like I can just drop this hunk.